e2e/icons-react/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -1130,12 +1168,14 @@ Array [
   "LogoVmwareAlt",
   "LogoVue",
   "LogoWechat",
+  "LogoX",
   "LogoXing",
   "LogoYelp",
   "LogoYoutube",
   "Logout",
   "Loop",
   "LowSeverity",
+  "Mac",
```

Comment:
Same with this, does it matter it isn't uppercase?

Reply:
here MAC stands for Media Access Control so it should probably be capitalized

Reply:
@laurenmrice actually I'm looking through some of the other icon names that are abbreviations and it looks like they all have the same treatment of first letter upper case and the rest lower case so maybe it's fine to leave them all alone